modules/stitching/include/opencv2/stitching.hpp

```diff
@@ -299,8 +299,13 @@ class CV_EXPORTS_W Stitcher
      */
     CV_WRAP Status stitch(InputArrayOfArrays images, InputArrayOfArrays masks, OutputArray pano);
 
-    std::vector<int> component() const { return indices_; }
-    std::vector<detail::CameraParams> cameras() const { return cameras_; }
+    /** @brief Returns indeces of input images used in panorama stitching
+     */
+    CV_WRAP std::vector<int> component() const { return indices_; }
+
+    /** Returns estimated camera parameters for all stitched images
+     */
+    CV_WRAP std::vector<cv::detail::CameraParams> cameras() const { return cameras_; }
```

Comment:
As I can see `setTransform` method is not wrapped, it seems that is not possible to set `cameras_` from Python.

Reply:
There is issue in current generator implementation:
```
In file included from /mnt/Projects/Projects/opencv/modules/python/src2/cv2.cpp:88:
/mnt/Projects/Projects/opencv-build/modules/python_bindings_generator/pyopencv_generated_types_content.h: In function ‘PyObject* pyopencv_cv_Stitcher_setTransform(PyObject*, PyObject*, PyObject*)’:
/mnt/Projects/Projects/opencv-build/modules/python_bindings_generator/pyopencv_generated_types_content.h:21351:5: error: ‘vector_detail_CameraParams’ was not declared in this scope; did you mean ‘vector_CameraParams’?
21351 |     vector_detail_CameraParams cameras;
```
Let's handle it separately.